std_msgs/Header header
float64[49] mass_matrix
float64[42] jacobian_matrix